src/main/java/com/github/lzyzsd/circleprogress/DonutProgress.java

```diff
@@ -84,6 +99,34 @@ public void setProgress(float progress) {
 		}
 	}
 
+	public void setProgress(long bytesRead, long totalBytes) {
+
+		if(this.internalState != Type.SIZE_PROGRESSBAR) {
+
+			mbStatus = new Paint();
+			mbStatus.setColor(finishedStrokeColor);
+			mbStatus.setStyle(Paint.Style.FILL);
+			mbStatus.setStrokeWidth(unfinishedStrokeWidth);
+			mbStatus.setTextSize(40);
```

Comment:
This needs to be scaled for dp/sp. I probably would have just used a separate `TextView` rather than drawing the text directly, but this works too.